CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Respect stack components on different hosts when connecting to event sources in the Console.
```

Comment:
nit; changelog is in past tense

Reply:
No it's not. See https://keepachangelog.com/en/1.0.0/

Reply:
Hahah now I pushed a change already.